works.css

```diff
@@ -161,6 +163,75 @@ h1 {
   margin-left: 5px;
 }
 
-ul {
-  list-style-type: none;
+.language-name {
+  padding-top: 20px;
+}
+
+.bt {
+  display: flex;
+}
+
+#contact-me {
+  display: flex;
+  flex-direction: column;
+  background-color: #6070ff;
+  align-items: center;
+  justify-content: center;
+  color: white;
+  border-top-left-radius: 50px;
+  padding: 1rem;
+  background-image: url("./assets/contact\ form\ shapes-mobile.svg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: initial;
+}
+
+#contact-head {
+  margin-top: 8rem;
+  font-weight: bold;
+  font-size: 40px;
+}
+
+#contact-description {
```

Comment:
[OPTIONAL]

- Avoid using `Id` much.

Reply:
I can see that you reduced the use of ID's. well done!